reactDashboard/src/components/DataDefinitions/DataDefinitions.css

```diff
@@ -59,6 +59,22 @@
   color: #2a84bb;
 }
 
+.darkmode .card-body {
+  background-color: #121212;
+}
+
+.darkmode hr {
+  background-color: #f2f2f2;
+}
+
+.darkmode .accordion-header {
```

Comment:
I'd move this up to just below the .accordion-header clause - keep light and dark together